compiler/crates/relay-typegen/tests/generate_typescript/fixtures/updatable-fragment-spread-and-regular-spread.expected

```diff
@@ -0,0 +1,48 @@
+==================================== INPUT ====================================
+query updatableFragmentSpreadAndRegularSpreadQuery {
+	me {
+		...updatableFragmentSpreadAndRegularSpread_updatable_user
+		...updatableFragmentSpreadAndRegularSpread_user
+	}
+}
+
+fragment updatableFragmentSpreadAndRegularSpread_updatable_user on User @updatable {
+	firstName
+}
+
+fragment updatableFragmentSpreadAndRegularSpread_user on User {
+	firstName
+}
+==================================== OUTPUT ===================================
+import { FragmentRefs } from "relay-runtime";
+export type updatableFragmentSpreadAndRegularSpreadQuery$variables = {};
+export type updatableFragmentSpreadAndRegularSpreadQuery$data = {
+  readonly me: {
+    readonly $updatableFragmentSpreads: FragmentRefs<"updatableFragmentSpreadAndRegularSpread_updatable_user">;
+    readonly " $fragmentSpreads": FragmentRefs<"updatableFragmentSpreadAndRegularSpread_user">;
+  } | null;
```

Comment:
In the Flow version this is object `?` instead of `| null`. I believe `?` also includes `undefined`. In general, values read out of the store can always be `null` (server said they were null) or `undefined` (value was missing from the store).

I wonder if this is an existing imperfection in the TypeScript types?

Reply:
If the store can also return undefined, then I'd say the types should definitely reflect that. I've added the necessary change: a0f14566813e66dfb3e8ac36a90c59dada1789c6 Might be too big for this PR though, so maybe a follow-up?